docs/cvl/invariants.md

```diff
@@ -245,20 +263,21 @@ method:
 invariant balance_is_0(address a)
     balanceOf(a) == 0
     filtered {
-        f -> f.selector != deposit(uint).selector
+        f -> f.selector != sig:deposit(uint).selector
     }
 ```
 
 In this example, when the variable `f` is bound to `deposit(uint)`, the
-expression `f.selector != deposit(uint).selector` evaluates to `false`, so the
+expression `f.selector != sig:deposit(uint).selector` evaluates to `false`, so the
 method will be skipped.
 
 ```{note}
-If there is a {ref}`preserved block <preserved>` for a method it will be
-verified even if it should be filtered out.
+If there is a {ref}`preserved block <preserved>` for a method, the method will
```

Comment:
Oh, good that I asked. This is very different from what I originally understood. So the verification logic is something like this?
```
fun doVerify(method m) {
  if (hasSpecialPreservedBlock(m)) verify(m);
  if (evaluateFilter(m)) verify(m);
  skip(m);
}
```
I wonder whether we should at least throw a warning here ("invariant filter for <i> excluded method <m>, but a specialized preserved block was defined"). I'd argue that one way or another, the user is confused or forgot to delete something :-)